helmfile.d/stacks/monitoring-prometheus.yaml

```diff
@@ -14,9 +14,11 @@ templates:
     needs:
       - monitoring/networkpolicy
       - monitoring/podsecuritypolicy
+      {{- if .Values | get "ck8sManagementCluster.enabled" false }}
```

Comment:
Sounds like [`needs`](https://helmfile.readthedocs.io/en/latest/#dag-aware-installationdeletion-ordering-with-needs) is primarily about ordering, are these conditionals needed at all?

Reply:
From testing it does not seem like this check is actually needed so I will remove them

Reply:
Nvm, seems like our unit tests complain if it is listed in needs but is not installed :thinking: 
@aarnq as you wrote [these tests](https://github.com/elastisys/compliantkubernetes-apps/blob/main/tests/unit/templates/releases.bash), do you have any input about us changing the conditionals for `needs`?

Reply:
AFAIK, needs in helmfile is not just for ordering, it will check that the needed helm releases are installed before it install this release.

Reply:
It did seem like helmfile ignored the release for `wc` although it was included in needs but it did not follow the `condition`, but would need to do more testing to ensure this, I also feel like it would be better to make it work with our current tests, and then potentially look into this more in the future if we feel that we need to.